script.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+//1. Given an array consisting of movie names, iterate over the array with the output of the names of each .movie to the console.
+const movieNames = ["one","two","three","four","five","six","seven","eight","nine","ten"];
+for (let i =0; i<movieNames.length; i++) {
+    console.log(movieNames[i]);
+};
+
+
+
+//2.Given an array of car manufacturers, convert the array to a string and back to an array
+let carManufac = ["Toyota", "Ford", "Honda"];
+console.log(carManufac.join().split());
+
```

Comment:
check please, how split method works
the result here [ 'Toyota,Ford,Honda' ] it's not the same array as before